### DISCUSSION TITLE
Start/End Tour location counting sync with selected user

### DISCUSSION THREAD

**Discussion by mff73** [mytourbook/mytourbook#1247] [Ideas]
With upcoming 24.1 version, please consider linking the count usage of each location with the selected user
See here
https://github.com/mytourbook/mytourbook/issues/878#issuecomment-1871374575
And here
https://github.com/mytourbook/mytourbook/issues/878#issuecomment-1871744608

**Comment by wolfgang-ch** [comment]
Created an issue https://github.com/mytourbook/mytourbook/issues/1248

**Comment by wolfgang-ch** [comment]
This feature will be available in 24.1.

Not only the usage for a location is synched with the user but also the tour locations view is filtered and will display only locations for the selected user
